catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/localization/BricksCategoriesTest.java

```diff
@@ -0,0 +1,176 @@
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.catrobat.catroid.uiespresso.localization;
+
+import android.support.test.espresso.Espresso;
+import android.support.test.espresso.ViewInteraction;
+import android.support.test.espresso.action.ViewActions;
+import android.support.test.runner.AndroidJUnit4;
+
+import org.catrobat.catroid.ProjectManager;
+import org.catrobat.catroid.R;
+import org.catrobat.catroid.content.Project;
+import org.catrobat.catroid.content.Sprite;
+import org.catrobat.catroid.ui.ProgramMenuActivity;
+import org.catrobat.catroid.uiespresso.util.BaseActivityInstrumentationRule;
+import org.catrobat.catroid.uiespresso.util.UiTestUtils;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static android.support.test.espresso.assertion.LayoutAssertions.noEllipsizedText;
+import static android.support.test.espresso.assertion.LayoutAssertions.noOverlaps;
+import static android.support.test.espresso.assertion.ViewAssertions.matches;
+import static android.support.test.espresso.matcher.ViewMatchers.isCompletelyDisplayed;
+import static android.support.test.espresso.matcher.ViewMatchers.isRoot;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static android.support.test.espresso.matcher.ViewMatchers.withText;
+
+import static org.catrobat.catroid.uiespresso.localization.assertions.LayoutDirectionAssertions.isLayoutDirectionRTL;
+import static org.catrobat.catroid.uiespresso.localization.assertions.TextDirectionAssertions.isTextDirectionRTL;
+import static org.catrobat.catroid.uiespresso.localization.assertions.VisibilityAssertions.isVisible;
+import static org.hamcrest.core.IsNull.notNullValue;
+
+@RunWith(AndroidJUnit4.class)
+
+public class BricksCategoriesTest {
+	private ViewInteraction brickCategoryEvent;
+	private ViewInteraction brickCategoryControl;
+	private ViewInteraction brickCategorySound;
+	private ViewInteraction brickCategoryMotion;
+	private ViewInteraction brickCategoryLooks;
+	private ViewInteraction brickCategoryData;
+	private ViewInteraction brickCategoryPen;
+
+	@Rule
+	public BaseActivityInstrumentationRule<ProgramMenuActivity> baseActivityTestRule = new
+			BaseActivityInstrumentationRule<>(ProgramMenuActivity.class, true, false);
+
+	@Before
+	public void setUp() throws Exception {
+		UiTestUtils.createProject("BricksCategoriesTest");
+		baseActivityTestRule.launchActivity(null);
+		Espresso.onView(withId(R.id.program_menu_button_scripts)).perform(ViewActions.click());
```

Comment:
if you need the scriptActivity, then start it with the Rule instead of navigating there